src/components/List/List.tsx

```diff
@@ -61,23 +52,34 @@ export const List = <Item extends Entity>(props: ListProps<Item>) => {
     LoadMoreButtonProps,
     scrollRef: scrollRefProp,
     className,
+    sx,
+    containerSx,
   } = props;
-  const { classes, cx } = useStyles(props, {
-    props: { classes: props.classes },
-  });
 
   const scrollRef = useRef<HTMLDivElement | null>(null);
   usePersistedScroll(scrollRefProp ?? scrollRef);
   const determineChangesetDiff = useDetermineChangesetDiffItem();
 
   return (
-    <div className={cx(classes.root, className)} ref={scrollRef}>
+    <Box
+      component="div"
+      className={className}
+      ref={scrollRef}
+      css={(theme) => [
+        {
+          overflow: 'auto',
+          marginLeft: theme.spacing(-2),
+          padding: theme.spacing(2),
+        },
+      ]}
+      sx={containerSx}
+    >
       <Grid
         direction="column"
         spacing={spacing}
         {...ContainerProps}
         container
-        className={classes.container}
+        sx={sx}
```

Comment:
Isn't it a bit confusing to have `ContainerProps.sx` and `sx` and then `containerSx` which doesn't go to the "container"?

What if `sx` goes to the Box above and then `containerSx` just gets dropped in favor of `ContainerProps.sx`?

Reply:
The reason `sx` was added was because `List` component does not have and `sx` and does not inherit it either. `containerSx` will be removed :)